lessons/en/basics/functions.md

```diff
@@ -402,9 +402,9 @@ defmodule Greeter do
   def hello(names, language_code \\ "en")
 
   def hello(names, language_code) when is_list(names) do
-    names = Enum.join(names, ", ")
+    binary_names = Enum.join(names, ", ")
```

Comment:
I think this is a good change overall to avoid reassigning `names` but I'm not sure `binary_names` is actually much clearer, maybe we can go with something a bit more verbose but exact? `comma_separated_names`? `joined_names`?